index.html

```diff
@@ -263,7 +430,8 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
                 </div>
                 <div class="line-foot"></div>
             </footer>
-            
+            <!-- FOOTER SECTION END HERE -->
+
         </main>
 
 </body>
```

Comment:
- [ ] Based on the feedback from the previous reviewer, I noticed that some of your projects are **being hidden within the work section of your project**. To ensure that your work is properly displayed and easily accessible to your audience, you may want to make some adjustments. 🔍
  It seems that the issue with the hidden projects **begins to resolve at a viewport width of `1180px` and above.** One potential cause of this problem is the **fixed width that you have set for each project card**. Instead of using a` fixed width`, you may want to **consider using a responsive layout that adjusts to the size of the viewport**. 🤓

  ### Your design(Without Fixed Width):
  ![image](https://user-images.githubusercontent.com/107952724/236362043-6032405a-cedf-40da-89f4-298af39047fd.png)

  ----
  #### HINT: You might consider adjusting the number of columns and some of the grid properties, as well as the width. Additionally, adding a bit of margin between the items can help create a more visually appealing layout.
  ![image](https://user-images.githubusercontent.com/107952724/236363039-0865ad3a-e4d3-4f6f-92b7-f92312d06175.png)

  ----
  _The previous reviewer has attached a video that showcases the exact problem that needs to be addressed. Please take a moment to review the video and make the necessary adjustments to your project._ 🙋‍♂️